pages/initiatives/gsoc/gsoc2021ideas.md

```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
 * [Fred Blaise](mailto:fred.blaise@gmail.com )
 
 ----
+
+### [OWASP Bug Logging Tool](https://github.com/OWASP/BLT)
+
+##### Brief Explanation:
+
+BLT lets anyone report issues they find on the internet. Found something out of place on Amazon.com ? Let them know. Companies are held accountable and shows their response time and history. Get points for reporting bugs and help keep the internet bug free.
+
+Check OWASP WIKI PAGE https://www.owasp.org/index.php/OWASP_Bug_Logging_Tool for some reference;
```

Comment:
This needs updating

Reply:
Seems you need to actuallynuodate your project page too: https://owasp.org/www-project-bug-logging-tool/

Reply:
Removed this reference. Github project page is referenced with the title. Will update the OWASP project page soon.